docs/index.rst

```diff
@@ -6,6 +6,17 @@
 Welcome to Earthpy's documentation!
 ===================================
 
+Earthpy's documentation contains one Element and two Options.
```

Comment:
@mcosmos12 i'm not sure what the goal of this text is. This page is a landing page for the user and might typically describe what earthpy is. I suggest that you remove this change and instead try to document a function that has not yet been documented! 

It is also failing a build right now due to a spacing issue

[see here](https://travis-ci.org/earthlab/earthpy/builds/447594030#L2078)
But the bigger issue is you should update documentation for a function or provide an overview here that enhances the documentation.

[here is what rasterio's landing page looks like for reference.](https://rasterio.readthedocs.io/en/latest/) 
If i am missing something please let me know but I can not merge this as is! 
Thank you!

Reply:
@mcosmos12 @lwasser I believe that Megan plans to attend my office hours today and we can discuss an alternative at that time. Thanks.